python/Ganga/new_tests/GPI/GangaUnitTest.py

```diff
@@ -125,9 +125,15 @@ def stop_ganga():
 
         from Ganga.GPI import jobs, templates
         for j in jobs:
-            j.remove()
+            try:
+                j.remove()
+            except:
+                pass
         for t in templates:
-            t.remove()
+            try:
+                t.remove()
+            except:
+                pass
```

Comment:
In principle yes, however,
Do we care if the pull-down method in the test is failing as it shouldn't cause the test to fail should it?
Besides all we're doing is effectively an ```rm -fr ~/gangadir_testing``` so I don't think that it overly matters if the teardown fails due to something going wrong here. We could work out what the specific error type should be and wrap job.remove but I would say this only becomes important when testing the job.remove and we don't want to confuse test errors due to unrelated code failing imo.